### PR TITLE
Create a message package

### PR DESCRIPTION
**Pull request by agalbachicar**
# 🎉 New feature

Part of #1

## Summary

Creates `maliput_ros_interfaces` with some type definitions.

## Test it

Check CI.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Comment by agalbachicar** [comment]
The first CI check (push) failed because the repository was private. After making it public it is successful.

File: maliput_ros_interfaces/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,58 @@
+##############################################################################
+# Project
+##############################################################################
+
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+project(maliput_ros_interfaces)
```
Comment (francocipollone) [inline maliput_ros_interfaces/CMakeLists.txt:6]: nit: Simply nit to think/define naming.

If it is `maliput_ros_interfaces` then the next package that will integrate maliput with ros will be called: `maliput_ros`, right?

Should we rename the repo to be `maliput_ros_pkgs`?

File: maliput_ros_interfaces/msg/LaneId.msg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Represents a maliput::api::LaneId.
+string id
```
Comment (francocipollone) [inline maliput_ros_interfaces/msg/LaneId.msg:2]: In general if we only were to have Ids won't make sense to have a custom message with a sole string field. But given we expect to add several messages then makes sense to differentiate a String msg from a LaneId (or other msgs).

**Review by francocipollone — APPROVED** [review]
LGTM :rocket:

File: maliput_ros_interfaces/CMakeLists.txt (hunk 1 above)
Comment (agalbachicar) [reply maliput_ros_interfaces/CMakeLists.txt:6]: In my head, I have the following packages:

- maliput_ros_interfaces: message definition.
- maliput_ros_translation: message translation between ROS and maliput.
- maliput_ros: this one will have the query service manager, the node(s) and the launch configuration.

I think we can rename repo at any point, yes. I don't mind leaving it as ros2_maliput either.

File: maliput_ros_interfaces/msg/LaneId.msg (hunk 2 above)
Comment (agalbachicar) [reply maliput_ros_interfaces/msg/LaneId.msg:2]: Yes, I did that on purpose to cast them further to each entity type.

**Pull request merged**